setup.py

```diff
@@ -0,0 +1,33 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+requirements = ["celery[redis]==4.3.1"]
```

Comment:
Shouldn't we use the file requirements/base.txt here? 

We might change the requirements and forget to update it here, this way.

Reply:
Agreed! But I would use the base.in file instead. I left as simple as possible for this first Pull Request since the test suite is still not decide. I will code the import for requirements :D